wzdx/sample_files/raw/planned_events/process_planned_events.py

```diff
@@ -0,0 +1,90 @@
+import datetime
```

Comment:
This seems like we were using it to test with icone smart boards, should we move it to a better directory than sample_files?

Reply:
Still in sample_files directory.

Reply:
removed, these are no longer necessary